Formula/qgis2.rb

```diff
@@ -5,14 +5,14 @@ class Qgis2 < Formula
   desc "Open Source Geographic Information System"
   homepage "http://www.qgis.org"
 
-  head "https://github.com/qgis/QGIS.git", :branch => "master_2"
+  head "https://github.com/qgis/QGIS.git", :branch => "release-2_18"
```

Comment:
Why the switch from `master_2` to `release-2_18` for the `head` build? That's what the `devel` build block is for in this release branch formula.